docs/source/multidc/multidc.md

```diff
@@ -0,0 +1,563 @@
+                scylla/cluster: scylla-cluster
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: topology.kubernetes.io/zone
+                operator: In
+                values:
+                - us-east-1c
+              - key: scylla.scylladb.com/node-type
+                operator: In
+                values:
+                - scylla
+        tolerations:
+        - key: role
+          operator: Equal
+          value: scylla-clusters
+          effect: NoSchedule
+```
+
+Apply the manifest:
+```shell
+kubectl --context="${CONTEXT_DC1}" apply --server-side -f=dc1.yaml
+```
+
+Wait for the cluster to be fully rolled out:
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla wait --for='condition=Progressing=False' scyllaclusters.scylla.scylladb.com/scylla-cluster
+```
+```console
+scyllacluster.scylla.scylladb.com/scylla-cluster condition met
+```
+
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla wait --for='condition=Degraded=False' scyllaclusters.scylla.scylladb.com/scylla-cluster
+```
+```console
+scyllacluster.scylla.scylladb.com/scylla-cluster condition met
+```
+
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla wait --for='condition=Available=True' scyllaclusters.scylla.scylladb.com/scylla-cluster
+```
+```console
+scyllacluster.scylla.scylladb.com/scylla-cluster condition met
+```
+
+You can now verify that all the nodes of your cluster are in UN state:
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla exec -it pod/scylla-cluster-us-east-1-a-0 -c=scylla -- nodetool status
+```
+
+The expected output should look similar to the below:
+```console
+Datacenter: us-east-1
+=====================
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address      Load       Tokens       Owns    Host ID                               Rack
+UN  10.0.70.195  290 KB     256          ?       494277b9-121c-4af9-bd63-3d0a7b9305f7  c
+UN  10.0.59.24   559 KB     256          ?       a3a98e08-0dfd-4a25-a96a-c5ab2f47eb37  b
+UN  10.0.19.237  107 KB     256          ?       64b6292a-327f-4128-852a-6004039f402e  a
+```
+
+##### Retrieve PodIPs of ScyllaDB nodes for use as external seeds
+
+``` warning::
+    Due to the ephemeral nature of PodIPs, it is ill-advised to use them as seeds in production environments. 
+    This is because there is a high likelihood that the Pods of your ScyllaDB clusters will change their IPs during the cluster's lifecycle, and so the provided seeds will no longer point to the ScyllaDB nodes.
+    It is undesired, as the seeds provided on node's startup may serve as fallback contact points when all of the node's peers are unreachable.
+    PodIPs are being used in this example for the sheer simplicity of this setup.
+    In production environments, it is recommended that you use domain names or non-ephemeral IP addresses as external seeds.
```

Comment:
I would swap these two sententces. First recommend, then explain why we are not following the recommendation.

Reply:
done